src/renderer/common/js/utils.js

```diff
@@ -151,6 +151,21 @@
         return false;
     };
 
+    /**
+     * A custom function for handling executing the Snipping Tool command.
+     * @param {String} command - path to executable file
+     * @return {Boolean} - returns `true` on successfully executed command
+     */
+    gpii.windows.openSnippingTool = function (command) {
```

Comment:
Can't we just use gpii.windows.launchExecutable for this?

Reply:
I'm not sure the reason, but this "command" don't work if its escaped by quotes (which we need for the longer paths like "Programs files" for example). And of course launchExecutable checks if the file exists (and have the proper rights), which we cannot use for this command.

So yeah, in reality we can use launchExecutable, but for that I need to add another argument in order for know if its executable file, or command. But that's basically the same as using different function.

Reply:
Please add a comment to the function giving a short explanation of this

Reply:
Done, the explanation is added as comment on the function.